flow/designs/asap7/mock-array/simulate.sh

```diff
@@ -1,32 +1,48 @@
 #!/usr/bin/env bash
+#
+# Executes Verilator to generate a VCD file based on simulation
+# simulate.cpp is set up to write the VCD file into the results directory
+#
 set -ex
 
-# allow this script to be invoked from any folder
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-BASE=$DIR/../..
 
-cd $DIR
+RESULTS_DIR="$FLOW_HOME/results/asap7/mock-array/base"
```

Comment:
If we have FLOW_HOME, we should have the other variables, why are we overriding them here?